src/main/java/com/flipkart/zjsonpatch/InPlaceApplyProcessor.java

```diff
@@ -147,11 +147,11 @@ public void replace(List<String> path, JsonNode value) {
             if (isNullOrEmpty(fieldToReplace) && path.size() == 1)
                 target = value;
             else if (parentNode.isObject())
-                ((ObjectNode) parentNode).put(fieldToReplace, value);
+                ((ObjectNode) parentNode).set(fieldToReplace, value);
```

Comment:
Why the change? Is this semantically equivalent?

Reply:
The method is long deprecated in jackson api

Reply:
Now that you commented, this change shouldn't be in this commit, it passed without me noticing, sorry. I will let you decide if this should pass or not